### ISSUE TITLE
"Refreshing Skin" every minute appearing in Logfile

### ISSUE THREAD

**Issue by emp-00** [MrJordanRoth/NS2Win#14]
I stumbled upon one more "small improvement" and got the solution from SilverAzide, see here: https://forum.rainmeter.net/viewtopic.php?p=210498#p210498

According to his feedback the NS2Win code section [cUpdate] is not a good way of implementing the "1 minute" refresh. I removed this section and only changed the Webparser update interval accordingly, see the forum discussion. This works fine and no more logfile entries are generated. Side-effect: The "Refresh" is not visible anymore thus the 1s meter restart where no values/text is shown is eliminated. This is fine for me but may be unexpected. Just check it out if you want to.

@MrJordanRoth : If you're interested in my small adjustments, let me know - I can of course share the code with you. I know I could/should probably create a fork but have never done that before...

This is how my latest revision looks like:
![image](https://user-images.githubusercontent.com/60714349/194727485-89d9d30b-e211-4f24-8dba-1a52939a828e.png)
![image](https://user-images.githubusercontent.com/60714349/194727432-be64388f-1911-42da-84d8-9f7e32dd8d8c.png)

**Comment by MrJordanRoth** [comment]
Awesome, I never looked more in depth to this. You can definitely fork, I'm not a GitHub person and this actually the first file I've probably put on here and didn't think anyone would use it. 

I hope I'm adding and making the changes appropriately as well!

Learning experience for all!!

**Comment by emp-00** [comment]
Here's my current revision with the discussed improvements, For the sake of simplicity I'm uploading the NS2Win.ini as a zip file here... Happy to discuss and open to more improvements. I'm thinking about synchronizing the skin time to server-time in order to only calling the pebble api once every 5 mins instead of every minute. But maybe this is too much for me - as of now I'm already very, very happy with all the improvements. Another idea would be adding the "X min ago" text which nightscout shows as well, but I think too much information on the gadget is not the best idea.

[NS2Win_V1.1.2.zip](https://github.com/MrJordanRoth/NS2Win/files/9740283/NS2Win_V1.1.2.zip)

**Comment by MrJordanRoth** [comment]
Hey, I'm ecstatic you found some use in the code to pull your glucose down. When I found this it was only working with Sugarmate - I changed it up so it could download from NightScout - the possibilities are endless.  I'm also using a locally hosted server and never thought twice about the time. That was a great addition and change - I will be watching and adding to the forums for the times. I think there is a good argument to implement something like that.
